### ISSUE TITLE
Automated Accessibility alerts for main pages

### ISSUE THREAD

**Issue by navaneethsnair1** [OpenLiberty/openliberty.io#4072]
It would be nice to run accessibility tests on our pages after every merge to prod to be ensured that we are compliant.

An issue can be created automatically if found any violations indicating in which page
